lambda-src/package.json

```diff
@@ -4,7 +4,14 @@
   "description": "Sends a message to a Slack channel",
   "main": "index.js",
   "scripts": {
-    "test": "mocha -u bdd --timeout 999999 --colors ../test"
+    "test": "npm run lint && npm run test:unit",
```

Comment:
I think I would prefer leaving linting and unit testing as 2 separate pipeline steps (I personally think it's nicer as a developer to see straight away if the pipeline failed because of  linting or unit testing issues) and therefore I would prefer if the test script didn't include lining

Reply:
Well `npm run test:unit` is still available for that workflow.

But if you don't like it I'm happy to approve the parent PR and we can close this.

Reply:
Oh actually there's still some other stuff we might want.

I'll push a commit to change this back.

Reply:
Pushed. How's that?

Reply:
That's great. Thanks Daniel